crates/accelerate/src/linear_matrix.rs

```diff
@@ -119,9 +119,23 @@ fn _gauss_elimination(
     let _perm = gauss_elimination_with_perm(view, ncols, full_elim);
 }
 
+#[pyfunction]
+#[pyo3(signature = (mat))]
+// Given a boolean matrix A after Gaussian elimination, computes its rank
+// (i.e. simply the number of nonzero rows)"""
+fn _compute_rank_after_gauss_elim(py: Python, mat: PyReadonlyArray2<bool>) -> PyResult<PyObject> {
+    let view = mat.as_array();
+    let rank: usize = view
+        .axis_iter(Axis(0))
```

Comment:
Just a tiny suggestion if you'd like to use it, I think this should be the same but shorter 🙂 
```suggestion
        .rows()
```

Reply:
thanks, but your suggestion raises the following compilation error:
error[E0599]: `Lanes<'_, bool, Dim<[usize; 1]>>` is not an iterator